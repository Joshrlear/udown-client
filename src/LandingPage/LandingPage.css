/* color scheme */
/* fonts */
.landing-page_container {
  position: absolute;
  display: block;
  width: 100vw;
  height: 92vh;
  font-family: "Poppins", sans-serif;
  line-height: 1.5em;
}

.landing-page_container .landing-page {
  position: relative;
  width: 100vw;
  height: 100vh;
}

@media screen and (min-width: 1100px) {
  .landing-page_container .landing-page {
    position: initial;
    display: -ms-grid;
    display: grid;
    grid: "title map" "title chat" "status info" / 55% 45%;
  }
}

.landing-page_container .title_section {
  height: 100vh;
}

@media screen and (min-width: 1100px) {
  .landing-page_container .title_section {
    grid-area: title;
  }
}

.landing-page_container .title_container {
  position: absolute;
  width: 100vw;
  top: calc(50% - 3em);
  -webkit-box-pack: center;
      -ms-flex-pack: center;
          justify-content: center;
  -webkit-box-align: center;
      -ms-flex-align: center;
          align-items: center;
  line-height: normal;
}

@media screen and (min-width: 1100px) {
  .landing-page_container .title_container {
    position: fixed;
    width: 55%;
  }
}

.landing-page_container .background_container {
  width: 100vw;
  height: 100vh;
  position: fixed;
  background: linear-gradient(104deg, #3cb4dc 30%, #c8db16 120%);
  overflow: hidden;
  z-index: -1;
}

@media screen and (min-width: 1100px) {
  .landing-page_container .background_container {
    width: 55vw;
  }
}

.landing-page_container .background_container .background {
  width: 100vw;
  height: 140vh;
  left: -8%;
  top: -5%;
  position: fixed;
  z-index: 0;
  mix-blend-mode: overlay;
  opacity: 0.4;
  -webkit-filter: blur(2.2px);
          filter: blur(2.2px);
}

.landing-page_container.off {
  display: none;
}

.landing-page_container .title, .landing-page_container .subtitle {
  text-align: center;
  color: white;
}

.landing-page_container .title {
  font-family: "Poppins", sans-serif;
  font-size: 56px;
  font-weight: 700;
}

.landing-page_container .subtitle {
  font-family: "Poppins", sans-serif;
  font-size: 20px;
  font-weight: 200;
  margin-top: -10px;
}

.landing-page_container .map_section {
  background-color: white;
  height: 90vh;
  margin-top: 15%;
}

@media screen and (min-width: 1100px) {
  .landing-page_container .map_section {
    grid-area: map;
  }
}

.landing-page_container .map_section .phone_container {
  margin: 0% 0% 15% 15%;
}

@media screen and (min-width: 1100px) {
  .landing-page_container .map_section .phone_container {
    position: initial;
    margin: 0% 0% 10% 0%;
    height: 50vh;
  }
}

.landing-page_container .map_section .infoDisplay, .landing-page_container .map_section .map {
  position: relative;
  width: 50%;
  -o-object-fit: contain;
     object-fit: contain;
}

.landing-page_container .map_section .map {
  top: 105%;
  left: 30%;
  -webkit-filter: blur(0.5px);
          filter: blur(0.5px);
}

@media screen and (min-width: 1100px) {
  .landing-page_container .map_section .map {
    height: 100%;
    top: initial;
    left: 28%;
    -webkit-filter: blur(0.75px);
            filter: blur(0.75px);
  }
}

.landing-page_container .map_section .infoDisplay {
  position: absolute;
  top: 105%;
  right: 35%;
}

@media screen and (min-width: 1100px) {
  .landing-page_container .map_section .infoDisplay {
    height: 100%;
    position: relative;
    top: 10%;
    right: 33%;
  }
}

.landing-page_container .map_section .textBox_container {
  display: -ms-grid;
  display: grid;
}

.landing-page_container .map_section .textBox_container h3 {
  margin: 5% auto 0% auto;
  font-weight: 300;
  line-height: 1.5em;
}

.landing-page_container .chat_section {
  position: relative;
  height: 100vh;
}

@media screen and (min-width: 1100px) {
  .landing-page_container .chat_section {
    grid-area: chat;
  }
}

.landing-page_container .chat_section .chat_whitespace {
  position: absolute;
  height: 48vh;
  width: 100vw;
  bottom: 0%;
  background-color: white;
}

@media screen and (min-width: 1100px) {
  .landing-page_container .chat_section .chat_whitespace {
    height: initial;
  }
}

.landing-page_container .chat_section .chat_main_container {
  height: 100%;
  display: -ms-grid;
  display: grid;
  position: relative;
  z-index: 0;
  -ms-grid-rows: (1fr)[9];
      grid-template-rows: repeat(9, 1fr);
  -ms-grid-columns: 5% 90% 5%;
      grid-template-columns: 5% 90% 5%;
}

.landing-page_container .chat_section .chat_main_container .chat_text_container {
  -ms-grid-row: 2;
      grid-row-start: 2;
  grid-row-end: 3;
  -ms-grid-column: 2;
      grid-column-start: 2;
  grid-column-end: 2;
  margin-top: 3vh;
  color: white;
}

@media screen and (min-width: 1100px) {
  .landing-page_container .chat_section .chat_main_container .chat_text_container {
    margin-top: 5vh;
    color: black;
  }
}

.landing-page_container .chat_section .chat_main_container .chat_img_container {
  -ms-grid-row: 3;
      grid-row-start: 3;
  grid-row-end: 9;
  -ms-grid-column: 1;
      grid-column-start: 1;
  grid-column-end: 4;
  z-index: 1;
}

@media screen and (min-width: 1100px) {
  .landing-page_container .chat_section .chat_main_container .chat_img_container {
    z-index: initial;
  }
}

.landing-page_container .chat_section .chat_main_container .chat_img_container .chat_img {
  height: 80%;
  -o-object-fit: contain;
     object-fit: contain;
}

.landing-page_container .chat_section .chat_main_container .chat_quote_container {
  -ms-grid-row: 8;
      grid-row-start: 8;
  grid-row-end: 9;
  -ms-grid-column: 2;
      grid-column-start: 2;
  grid-column-end: 3;
  z-index: 1;
  margin: auto;
  padding: 20px;
  border: rgba(0, 0, 0, 0.25) 2px solid;
  color: #1d2323;
}

@media screen and (min-width: 1100px) {
  .landing-page_container .chat_section .chat_main_container .chat_quote_container {
    z-index: initial;
  }
}

.landing-page_container .chat_section .chat_main_container .chat_quote_container .chat_quote {
  line-height: 1.5em;
}

.landing-page_container .map_msgs_section {
  display: -ms-grid;
  display: grid;
  overflow: hidden;
  height: 45vh;
  -ms-grid-columns: (1fr)[5];
      grid-template-columns: repeat(5, 1fr);
  -ms-grid-rows: 55% 45%;
      grid-template-rows: 55% 45%;
}

@media screen and (min-width: 1100px) {
  .landing-page_container .map_msgs_section {
    grid-area: status;
    position: relative;
    height: 100vh;
  }
}

.landing-page_container .map_msgs_section .status_map {
  height: 110%;
  -o-object-position: 42%;
     object-position: 42%;
  -ms-grid-column: 1;
      grid-column-start: 1;
  grid-column-end: 6;
  -ms-grid-row: 1;
      grid-row-start: 1;
  grid-row-end: 3;
}

.landing-page_container .map_msgs_section .status_msgs {
  padding-top: 15vh;
  width: 50vw;
  -ms-grid-column: 3;
      grid-column-start: 3;
  grid-column-end: 6;
  -ms-grid-row: 1;
      grid-row-start: 1;
  grid-row-end: 3;
}

@media screen and (min-width: 1100px) {
  .landing-page_container .map_msgs_section .status_msgs {
    width: 100%;
    padding-top: 30vh;
  }
}

.landing-page_container .status_section {
  background-color: white;
  height: -webkit-fit-content;
  height: -moz-fit-content;
  height: fit-content;
}

@media screen and (min-width: 1100px) {
  .landing-page_container .status_section {
    grid-area: info;
    padding: 15%;
  }
}

.landing-page_container .status_header_container {
  display: -ms-grid;
  display: grid;
  padding: 7%;
  -ms-grid-columns: (1fr)[3];
      grid-template-columns: repeat(3, 1fr);
  -ms-grid-rows: (1fr)[3];
      grid-template-rows: repeat(3, 1fr);
}

@media screen and (min-width: 1100px) {
  .landing-page_container .status_header_container {
    padding-top: 15%;
  }
}

.landing-page_container .status_header_container .status_project_title {
  font-size: 24px;
  -ms-grid-column: 1;
      grid-column-start: 1;
  grid-column-end: 3;
  -ms-grid-row: 1;
      grid-row-start: 1;
  grid-row-end: 4;
  -ms-flex-item-align: center;
      -ms-grid-row-align: center;
      align-self: center;
  text-align: left;
}

.landing-page_container .status {
  position: relative;
  bottom: -2px;
  -ms-grid-column: 3;
      grid-column-start: 3;
  grid-column-end: 3;
  -ms-grid-row: 2;
      grid-row-start: 2;
  grid-row-end: 2;
  -ms-flex-item-align: center;
      -ms-grid-row-align: center;
      align-self: center;
}

@media screen and (min-width: 1100px) {
  .landing-page_container .status {
    -ms-grid-column: 2;
        grid-column-start: 2;
    left: 10px;
  }
}

.landing-page_container .status_description_container {
  display: -ms-grid;
  display: grid;
  padding: 7%;
  padding-top: 0%;
  -ms-grid-columns: (1fr)[3];
      grid-template-columns: repeat(3, 1fr);
  -ms-grid-rows: 10% 90%;
      grid-template-rows: 10% 90%;
}

.landing-page_container .status_description_container .description_title {
  padding-bottom: 10px;
  -ms-grid-column: 1;
      grid-column-start: 1;
  grid-column-end: 1;
  -ms-grid-row: 1;
      grid-row-start: 1;
  grid-row-end: 1;
  -ms-flex-item-align: center;
      -ms-grid-row-align: center;
      align-self: center;
  color: grey;
}

.landing-page_container .status_description_container .description {
  -ms-grid-column: 1;
      grid-column-start: 1;
  grid-column-end: 4;
  -ms-grid-row: 2;
      grid-row-start: 2;
  grid-row-end: 3;
  -ms-flex-item-align: center;
      -ms-grid-row-align: center;
      align-self: center;
}

.landing-page_container .status_icon_links {
  height: 10vh;
  display: -webkit-box;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: space-evenly;
      -ms-flex-pack: space-evenly;
          justify-content: space-evenly;
  margin-left: 15%;
  margin-right: 15%;
}

@media screen and (min-width: 1100px) {
  .landing-page_container .status_icon_links {
    position: relative;
    top: 10%;
  }
}

.landing-page_container .status_icon_links .fab {
  color: black;
  -webkit-transition: all 0.15s ease;
  transition: all 0.15s ease;
}

.landing-page_container .status_icon_links .fab:hover {
  color: #c8db16;
}
/*# sourceMappingURL=LandingPage.css.map */