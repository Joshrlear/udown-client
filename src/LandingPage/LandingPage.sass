.landing-page_container 
    position: absolute
    display: block
    width: 100vw
    height: 100vh - 7vh

    &.off
        display: none

    .landing-page 
        position: relative
        top: calc(50% - 100px)

    .title 
        text-align: center
    
    .button_rack 
        display: flex
        justify-content: center
    
        .button_rack button 
            margin: 0px 8px
