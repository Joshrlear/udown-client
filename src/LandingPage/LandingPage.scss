@import '../utilities/_variables';

@mixin desktop() {
  @media screen and ( min-width: 1024px ) {
    @content;
  }
}

@mixin tablet() {
  @media screen and ( min-width: 760px ) {
    @content;
  }
}

@mixin l-tablet() {
  @media screen and ( min-width: 900px ) {
    @content;
  }
}

@mixin left {
    grid-area: left;
}

@mixin right {
    grid-area: right;
}



.landing-page_container {
    position: absolute;
    display: block;
    width: 100vw;
    height: 100vh - 8vh;
    font-family: $poppins-font;
    line-height: 1.5em;

    .landing-page {
        position: relative;
        width: 100vw;
        height: 100vh;
        @include desktop() {
            position: initial;
            display: grid;
            grid: "title map"
                  "title chat"
                  "status info"
                  / 55% 45%;
        }
    }


    .title_section {
        height: 100vh;
        @include desktop() {
            grid-area: title;
        }
    }
            


    .title_container {
        position: absolute;
        width: 100vw;
        top: calc(50% - 3em);
        justify-content: center;
        align-items: center;
        line-height: normal;
        @include desktop() {
            position: fixed;
            width: 55%;
        }
    }

    .background_container {
        width: 100vw;
        height: 100vh;
        position: fixed;
        background: linear-gradient(104deg, $blue-color 30%, $accent-color 120%);
        overflow: hidden;
        z-index: -1;
        @include desktop() {
            width: 55vw
        }

        .background {
            width: 100vw;
            height: 140vh;
            left: -8%;
            top: -5%;
            position: fixed;
            z-index: 0;
            mix-blend-mode: overlay;
            opacity: 0.4;
            filter: blur(2.2px);
        }
    }

    &.off {
        display: none;
    }

    .title, .subtitle {
        text-align: center;
        color: white;
    }

    .title {
        font-family: $poppins-font;
        font-size: 56px;
        font-weight: 700;
    }

    .subtitle {
        font-family: $poppins-font;
        font-size: 20px;
        font-weight: 200;
        margin-top: -10px;
    }

    .map_section {
        background-color: white;
        height: 90vh;
        margin-top: 10%;
        @include desktop() {
            grid-area: map;
        }
        
        .map_main_container {
            display: grid;
            height: 100%;
            grid-template-rows: repeat(9, 1fr);
        }
        

        .phone_container {
            grid-row: 1/6;
            height: 100%;
            margin-top: 10%;
            @include desktop() {
                position: initial;
                height: 50vh;
            }
            @include tablet() {
                height: 100%;
            }
        }

        .infoDisplay, .map {
            position: relative;
            width: 50%;
            object-fit: contain;
        }

        .map {
            height: 70%;
            top: -15%;
            left: 35%;
            filter: blur(0.8px);
            @include desktop() {
                height: 100%;
                top: 4%;
                left: 28%;
                filter: blur(0.1vw);
            }
            @include tablet() {
                height: 90%;
                top: -5%;
                filter: blur(0.14vw);
            }
        }

        .infoDisplay {
            position: relative;
            height: 100%;
            right: 33%;
            top: initial;
            @include desktop() {
                position: relative;
                top: 1%;
            }
            @include tablet() {
                position: relative;
                height: 100%;
                right: 33%
            }
        }

        .textBox_container {
            display: grid;
            grid-row: 7/10;
            @include desktop() {
                height: 100%;
                align-items: center;
                grid-row: 6/9;
            }
            @include tablet() {
                margin-top: 5%;
            }

            h3 {
                margin: 5% auto 0% auto;
                font-weight: 300;
                line-height: 1.5em;
            }
        }
    }

    .chat_section {
        position: relative;
        height: 100vh;
        @include desktop() {
            grid-area: chat;
        }
            

        .chat_whitespace {
            position: absolute;
            height: 48vh;
            width: 100vw;
            bottom: 0%;
            background-color: white;
            @include desktop() {
                height: initial;
            }
        }

        .chat_main_container {
            height: 100%;
            display: grid;
            position: relative;
            z-index: 0;
            grid-template-rows: repeat(9, 1fr);
            grid-template-columns: 5% 90% 5%;

            .chat_text_container {
                grid-row-start: 2;
                grid-row-end: 3;
                grid-column-start: 2;
                grid-column-end: 2;
                margin-top: 3vh;
                color: white;
                @include desktop() {
                    margin-top: 5vh;
                    color: black;
                }
            }
                
            .chat_img_container {
                grid-row-start: 3;
                grid-row-end: 7;
                grid-column-start: 1;
                grid-column-end: 4;
                z-index: 1;
                overflow: hidden;
                @include desktop() {
                    grid-row: 3/8;
                    z-index: initial;
                }

                .chat_img {
                    height: 100%;
                    width: 130%;
                    padding: 5%;
                    object-fit: contain;
                    position: relative;
                    right: 15%;
                }
            }

            .chat_quote_container {
                grid-row-start: 7;
                grid-row-end: 9;
                grid-column-start: 2;
                grid-column-end: 3;
                z-index: 1;
                margin: auto;
                padding: 4vw;
                color: $darkGrey-color;
                @include desktop() {
                    grid-row: 7/10;
                    padding: 4%;
                    z-index: initial;
                }
                @include tablet() {
                    grid-row-start: 8;
                    grid-row-end: 9;
                    border: rgba(0, 0, 0, 0.25) 2px solid;
                }
                
                .chat_quote {
                    line-height: 2rem;
                }
            }
        }
    }

    .map_msgs_section {
        display: grid;
        overflow: hidden;
        height: 45vh;
        grid-template-columns: repeat(5, 1fr);
        grid-template-rows: 55% 45%;
        @include desktop() {
            grid-area: status;
            position: relative;
            height: 92vh;
        }
            

        .status_map {
            height: 110%;
            object-position: 42%;
            grid-column-start: 1;
            grid-column-end: 6;
            grid-row-start: 1;
            grid-row-end: 3;
        }
            
        .status_msgs {
            padding-top: 15vh;
            width: 50vw;
            grid-column-start: 3;
            grid-column-end: 6;
            grid-row-start: 1;
            grid-row-end: 3;
            @include desktop() {
                width: 100%;
                padding-top: 30vh;
            }
        }
    }

    .status_section {
        background-color: white;
        height: fit-content;
        @include desktop() {
            height: 100%;
            position: relative;
            grid-area: info;
            padding: 19% 15% 0% 15%;
        }
    }

        .status_header_container {
            display: grid;
            padding: 7%;
            grid-template-columns: 11.5rem;
            grid-template-rows: repeat(3, 1fr);
            

            .status_project_title {
                font-size: 24px;
                grid-column-start: 1;
                grid-column-end: 3;
                grid-row-start: 1;
                grid-row-end: 4;
                align-self: center;
                text-align: left;
            }
        }
                
            .status {
                position: relative;
                bottom: -2px;
                grid-column-start: 2;
                grid-column-end: 3;
                grid-row-start: 2;
                grid-row-end: 2;
                align-self: center;
                @include desktop() {
                    left: 10px;
                }
            }

        .status_description_container {
            display: grid;
            padding: 7%;
            padding-top: 0%;
            grid-template-columns: repeat(3, 1fr) ;
            grid-template-rows: 10% 90%;

            .description_title {
                padding-bottom: 10px;
                grid-column-start: 1;
                grid-column-end: 1;
                grid-row-start: 1;
                grid-row-end: 1;
                align-self: center;
                color: grey;
            }

            .description {
                grid-column-start: 1;
                grid-column-end: 4;
                grid-row-start: 2;
                grid-row-end: 3;
                align-self: center;
            }
        }

        .status_icon_links {
            height: 10vh;
            display: flex;
            justify-content: space-evenly;
            margin-left: 15%;
            margin-right: 15%;
            @include desktop() {
                position: relative;
                padding-top: 5%
            }

            .fab {
                color: black;
                transition: all 0.15s ease;
                

                &:hover {
                    color: $accent-color;
                }
            }
        }
}