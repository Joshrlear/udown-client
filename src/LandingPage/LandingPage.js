import React from 'react';
import UdownContext from '../UdownContext';
import './LandingPage.css';
import background from './images/tennisGuyHappy.png'
import infoDisplay from './images/infoDisplay@300x.png'
import map from './images/map@300x.png'
import chatToNav from './images/chatToNav.png'
import statusMap from './images/desktop_map-icons-01.png'
import msgs from './images/phone_msgs.png'

export default class LandingPage extends React.Component {

    static contextType = UdownContext;

    render() {
        const containerClass = !this.context.chatOpened ? 'landing-page_container ' : 'landing-page_container off'

        return (
            <div className={ containerClass }>
                <div className="landing-page">
                    <section className="title_section">
                        <div className="title_container">
                            <h1 className="title">udown</h1>
                            <h3 className="subtitle">anytime | anywhere</h3>
                            <a className="demo_link" href="https://youtu.be/B_JTZHo4pgs" target="_blank" rel="noopener noreferrer">See Demo</a>
                        </div>
                        <div className="background_container">
                            <img alt="excited tennis player" className="background" src={ background }/>
                        </div>
                    </section>
                    <section className="map_section">
                        <div className="map_main_container">
                            <div className="phone_container">
                                <img alt="blurred phone displaying map" className="map" src={ map }/>
                                <img alt="phone displaying info of chosen location" className="infoDisplay" src={ infoDisplay }/>
                            </div>
                            <article className="textBox_container">
                                <h3>
                                    udown helps you plan<br/>
                                    events, matches, and<br/>
                                    hangouts now, not later.
                                </h3>
                            </article>
                        </div>
                    </section>
                    <section className="chat_section">
                        <div className="chat_whitespace"/>
                        <div className="chat_main_container">
                            <header className="chat_text_container">
                                <h2 className="chat_text">
                                    Ready. Set. Go!
                                </h2>
                            </header>
                            <div className="chat_img_container">
                                <img alt="phone with three floating screens showing process from text message, to chat, to navigation" className="chat_img" src={ chatToNav }/>
                            </div>
                            <article className="chat_quote_container">
                                <p className="chat_quote">
                                    <em>
                                        "...there really isn't anything<br/>
                                        that brings us together in<br/>
                                        the real world... so I built it."
                                    </em>
                                </p>
                            </article>
                        </div>
                    </section>
                        <section className="map_msgs_section">
                            <img alt="map displaying users and their events" className="status_map" src={ statusMap }/>
                            <img alt="phone displaying chat feature" className="status_msgs" src={ msgs }/>
                        </section>
                        <section className="status_section">
                            <header className="status_header_container">
                                <h2 className="status_project_title">Project udown</h2>
                                <p className="status">| active</p>
                            </header>
                            <article className="status_description_container">
                                <p className="description_title">Description</p>
                                <p className="description">
                                    Do you like planning far out or hate other
                                    people? Then this app is likely not for you
                                    <br/>
                                    <br/>
                                    For those of you who answered no to the
                                    previous question, join us on the platform
                                    built to bring soon to be "besties" together
                                    <br/>
                                    <br/>
                                    udown is a (soon to be) cross-platform app
                                    that connects locasls who are currently, like
                                    right now, looking for others to play tennis,
                                    basketball, go to the gym, bar, or even
                                    protest their current cause with.
                                    <br/>
                                    <br/>
                                    For video on how to use this website click the 
                                    YouTube icon.
                                </p>
                            </article>
                            <footer className="status_icon_links">
                                <a href="https://github.com/Joshrlear/udown-client"><span className="fab fa-github-square fa-2x"/></a>
                                <a href="https://www.facebook.com/josh.lear.7"><span className="fab fa-facebook-square fa-2x"/></a>
                                <a href="https://youtu.be/B_JTZHo4pgs" target="_blank" rel="noopener noreferrer"><span className="fab fa-youtube-square fa-2x"/></a>
                            </footer>
                        </section>
                </div>
            </div>
        )
    }
}