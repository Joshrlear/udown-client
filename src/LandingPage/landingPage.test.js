import * as React from 'react';
import ReactDOM from 'react-dom';
import { BrowserRouter } from 'react-router-dom';
import { shallow, configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import LandingPage from './LandingPage';

configure({ adapter: new Adapter() })

describe('LandingPage rendering', () => {
    it(`renders LandingPage without crashing`, () => {
        const div = document.createElement('div');
        ReactDOM.render(
        <BrowserRouter>
           <LandingPage/>
        </BrowserRouter>, div)
        ReactDOM.unmountComponentAtNode(div)
    })

    const resizeWindow = (x, y) => {
        window.innerWidth = x;
        window.innerHeight = y;
        window.dispatchEvent(new Event('resize'));
      }
      
    it('should display the window size', () => {
        const component = shallow(<LandingPage />);

        resizeWindow(320, 568);
        expect(component.html()).toMatchSnapshot();

        resizeWindow(1024, 1366);
        expect(component.html()).toMatchSnapshot();

        resizeWindow(1920, 1080);
        expect(component.html()).toMatchSnapshot();
    });
})