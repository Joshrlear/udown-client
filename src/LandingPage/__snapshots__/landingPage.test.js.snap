// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LandingPage rendering should display the window size 1`] = `"<div class=\\"landing-page_container \\"><div class=\\"landing-page\\"><section class=\\"title_section\\"><div class=\\"title_container\\"><h1 class=\\"title\\">udown</h1><h3 class=\\"subtitle\\">anytime | anywhere</h3></div><div class=\\"background_container\\"><img class=\\"background\\" src=\\"tennisGuyHappy.png\\"/></div></section><section class=\\"map_section\\"><div class=\\"map_main_container\\"><div class=\\"phone_container\\"><img class=\\"map\\" src=\\"map@300x.png\\"/><img class=\\"infoDisplay\\" src=\\"infoDisplay@300x.png\\"/></div><article class=\\"textBox_container\\"><h3>udown helps you plan<br/>events, matches, and<br/>hangouts now, not later.</h3></article></div></section><section class=\\"chat_section\\"><div class=\\"chat_whitespace\\"></div><div class=\\"chat_main_container\\"><header class=\\"chat_text_container\\"><h2 class=\\"chat_text\\">Ready. Set. Go!</h2></header><div class=\\"chat_img_container\\"><img class=\\"chat_img\\" src=\\"chatToNav.png\\"/></div><article class=\\"chat_quote_container\\"><p class=\\"chat_quote\\"><em>&quot;...there really isn&#x27;t anything<br/>that brings us together in<br/>the real world... so I built it.&quot;</em></p></article></div></section><section class=\\"map_msgs_section\\"><img class=\\"status_map\\" src=\\"desktop_map-icons-01.png\\"/><img class=\\"status_msgs\\" src=\\"phone_msgs.png\\"/></section><section class=\\"status_section\\"><header class=\\"status_header_container\\"><h2 class=\\"status_project_title\\">Project udown</h2><p class=\\"status\\">| active</p></header><article class=\\"status_description_container\\"><p class=\\"description_title\\">Description</p><p class=\\"description\\">Do you like planning far out or hate other people? Then this app is likely not for you<br/><br/>For those of you who answered no to the previous question, join us on the platform built to bring soon to be &quot;besties&quot; together<br/><br/>udown is a (soon to be) cross-platform app that connects locasls who are currently, like right now, looking for others to play tennis, basketball, go to the gym, bar, or even protest their current cause with.</p></article><footer class=\\"status_icon_links\\"><a href=\\"https://github.com/Joshrlear/udown-client\\"><span class=\\"fab fa-github-square fa-2x\\"></span></a><a href=\\"https://www.facebook.com/josh.lear.7\\"><span class=\\"fab fa-facebook-square fa-2x\\"></span></a><a href=\\"https://www.youtube.com/\\"><span class=\\"fab fa-youtube-square fa-2x\\"></span></a></footer></section></div></div>"`;

exports[`LandingPage rendering should display the window size 2`] = `"<div class=\\"landing-page_container \\"><div class=\\"landing-page\\"><section class=\\"title_section\\"><div class=\\"title_container\\"><h1 class=\\"title\\">udown</h1><h3 class=\\"subtitle\\">anytime | anywhere</h3></div><div class=\\"background_container\\"><img class=\\"background\\" src=\\"tennisGuyHappy.png\\"/></div></section><section class=\\"map_section\\"><div class=\\"map_main_container\\"><div class=\\"phone_container\\"><img class=\\"map\\" src=\\"map@300x.png\\"/><img class=\\"infoDisplay\\" src=\\"infoDisplay@300x.png\\"/></div><article class=\\"textBox_container\\"><h3>udown helps you plan<br/>events, matches, and<br/>hangouts now, not later.</h3></article></div></section><section class=\\"chat_section\\"><div class=\\"chat_whitespace\\"></div><div class=\\"chat_main_container\\"><header class=\\"chat_text_container\\"><h2 class=\\"chat_text\\">Ready. Set. Go!</h2></header><div class=\\"chat_img_container\\"><img class=\\"chat_img\\" src=\\"chatToNav.png\\"/></div><article class=\\"chat_quote_container\\"><p class=\\"chat_quote\\"><em>&quot;...there really isn&#x27;t anything<br/>that brings us together in<br/>the real world... so I built it.&quot;</em></p></article></div></section><section class=\\"map_msgs_section\\"><img class=\\"status_map\\" src=\\"desktop_map-icons-01.png\\"/><img class=\\"status_msgs\\" src=\\"phone_msgs.png\\"/></section><section class=\\"status_section\\"><header class=\\"status_header_container\\"><h2 class=\\"status_project_title\\">Project udown</h2><p class=\\"status\\">| active</p></header><article class=\\"status_description_container\\"><p class=\\"description_title\\">Description</p><p class=\\"description\\">Do you like planning far out or hate other people? Then this app is likely not for you<br/><br/>For those of you who answered no to the previous question, join us on the platform built to bring soon to be &quot;besties&quot; together<br/><br/>udown is a (soon to be) cross-platform app that connects locasls who are currently, like right now, looking for others to play tennis, basketball, go to the gym, bar, or even protest their current cause with.</p></article><footer class=\\"status_icon_links\\"><a href=\\"https://github.com/Joshrlear/udown-client\\"><span class=\\"fab fa-github-square fa-2x\\"></span></a><a href=\\"https://www.facebook.com/josh.lear.7\\"><span class=\\"fab fa-facebook-square fa-2x\\"></span></a><a href=\\"https://www.youtube.com/\\"><span class=\\"fab fa-youtube-square fa-2x\\"></span></a></footer></section></div></div>"`;

exports[`LandingPage rendering should display the window size 3`] = `"<div class=\\"landing-page_container \\"><div class=\\"landing-page\\"><section class=\\"title_section\\"><div class=\\"title_container\\"><h1 class=\\"title\\">udown</h1><h3 class=\\"subtitle\\">anytime | anywhere</h3></div><div class=\\"background_container\\"><img class=\\"background\\" src=\\"tennisGuyHappy.png\\"/></div></section><section class=\\"map_section\\"><div class=\\"map_main_container\\"><div class=\\"phone_container\\"><img class=\\"map\\" src=\\"map@300x.png\\"/><img class=\\"infoDisplay\\" src=\\"infoDisplay@300x.png\\"/></div><article class=\\"textBox_container\\"><h3>udown helps you plan<br/>events, matches, and<br/>hangouts now, not later.</h3></article></div></section><section class=\\"chat_section\\"><div class=\\"chat_whitespace\\"></div><div class=\\"chat_main_container\\"><header class=\\"chat_text_container\\"><h2 class=\\"chat_text\\">Ready. Set. Go!</h2></header><div class=\\"chat_img_container\\"><img class=\\"chat_img\\" src=\\"chatToNav.png\\"/></div><article class=\\"chat_quote_container\\"><p class=\\"chat_quote\\"><em>&quot;...there really isn&#x27;t anything<br/>that brings us together in<br/>the real world... so I built it.&quot;</em></p></article></div></section><section class=\\"map_msgs_section\\"><img class=\\"status_map\\" src=\\"desktop_map-icons-01.png\\"/><img class=\\"status_msgs\\" src=\\"phone_msgs.png\\"/></section><section class=\\"status_section\\"><header class=\\"status_header_container\\"><h2 class=\\"status_project_title\\">Project udown</h2><p class=\\"status\\">| active</p></header><article class=\\"status_description_container\\"><p class=\\"description_title\\">Description</p><p class=\\"description\\">Do you like planning far out or hate other people? Then this app is likely not for you<br/><br/>For those of you who answered no to the previous question, join us on the platform built to bring soon to be &quot;besties&quot; together<br/><br/>udown is a (soon to be) cross-platform app that connects locasls who are currently, like right now, looking for others to play tennis, basketball, go to the gym, bar, or even protest their current cause with.</p></article><footer class=\\"status_icon_links\\"><a href=\\"https://github.com/Joshrlear/udown-client\\"><span class=\\"fab fa-github-square fa-2x\\"></span></a><a href=\\"https://www.facebook.com/josh.lear.7\\"><span class=\\"fab fa-facebook-square fa-2x\\"></span></a><a href=\\"https://www.youtube.com/\\"><span class=\\"fab fa-youtube-square fa-2x\\"></span></a></footer></section></div></div>"`;
